public/app/features/backup/components/StorageLocations/StorageLocations.tsx

```diff
@@ -2,19 +2,26 @@ import React, { FC, useState, useEffect } from 'react';
 import { Column, Row } from 'react-table';
 import { logger } from '@percona/platform-core';
 import { IconButton, useStyles } from '@grafana/ui';
+import { AppEvents } from '@grafana/data';
+import { appEvents } from 'app/core/core';
```

Comment:
```suggestion
import { appEvents } from 'app/core/app_events';
```

So you can use the mock file already created,

Reply:
Thanks for the tip !